File: Solid Rectangle/printSolidRectangle.java
import java.util.Scanner;

class printSolidRectangle {
  public static void main(String[] args) {
    java.util.Scanner sc = new Scanner(System.in);
    System.out.println("Enter the number of rows : ");
    int rows = sc.nextInt();
    System.out.println("Enter the number of columns : ");
    int columns = sc.nextInt();

    for (int i=0; i<rows; i++) {
      for (int j=0; j<columns; j++) {
        System.out.print("*");
      }
      System.out.print("\n");
    }

    sc.close();
  }
}

File: Solid Rectangle/printSolidRectangle.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});

// Function to print star pattern.
function printSolidRectangle(r, c) {
  for (let i=0; i<r; i++) {
    let pattern = "";
    for (let j=0; j<c; j++) {
      pattern += "*";
    }
    console.log(pattern);
  }
}

// rows and columns from user and calling printSolidRectangle on them.
rl.question('Enter the number of rows : ', rows => {
  rl.question('Enter the number of columns : ', columns => {
    printSolidRectangle(Number(rows), Number(columns));
    rl.close();
  })
});

